### ISSUE TITLE
Package geometry Error

### ISSUE THREAD

**Issue by rene-schoene** [tud-cd/tudscr#78]
After updating to a newer version of TexLive, the `tudscr` package seems to be broken.
I'm getting the following error message when building with `latexmk -pdf`

```
1: section
1: chapter
1: chapter
1: section

! Package geometry Error: \loadgeometry: name `' undefined.

See the geometry package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.11 \begin{document}
```

for the MWE:

```tex
\documentclass[english]{tudscrreprt}
\iftutex
  \usepackage{fontspec}
\else
  \usepackage[T1]{fontenc}
  \usepackage[ngerman=ngerman-x-latest]{hyphsubst}
\fi
\usepackage{babel}
\usepackage{isodate}

\begin{document}

\maketitle

\end{document}
```

Setting the option `cdgeometry=false` (which I though could helped) did not change the behaviour.

Strangely, when "ignoring" the error (by pressing Enter multiple times, or entering `q` for batch mode), a document is built.
However, this workaround is neither nice nor viable for CI build processes.

**Comment by mrpiggi** [comment]
There will be a hotfix release till the end of this week

**Comment by mrpiggi** [comment]
fixed with 13e9f3207534f6de31e2f0ef6a75f242e8e799f2
